### DISCUSSION TITLE
Capacitor Plugin Support

### DISCUSSION THREAD

**Discussion by jack-hanlon** [electric-sql/pglite#217] [Ideas]
This would be a really nice alternative to SQLite for those who are using PosgreSQL cloud instances. It would be nice if when working with an offline mode, I could have a pglite db for Hybrid and Mobile apps. Then I wouldn't have to modify the schema when syncing between local and cloud

**Comment by Deliana90** [comment]
can this package replace https://github.com/capacitor-community/sqlite ?

**Comment by bil-ash** [comment]
I have a slightly different requirement, but I guess it can be clubbed with this one.
I would like to run NodeBB on cloudflare workers. However, cloudflare workers only supports sqlite db(cloudflare-d1) and none of the DBs supported by Nodebb(mongo, postgres, redis). So, if a new filesystem is introduced with sqlite a backend, my problem will be solved and so will @jack-hanlon 's problem.

**Comment by alterhuman** [comment]
This would be great.
